pkg/controller/pxcbackup/deadline_test.go

```diff
@@ -338,4 +342,50 @@ var _ = Describe("Suspended deadline", func() {
 		err = r.checkSuspendedDeadline(context.Background(), cluster, cr)
 		Expect(err).ToNot(HaveOccurred())
 	})
+
```

Comment:
i think we need another case that job has suspended condition with last transition time is older than suspendedDeadlineSeconds and `spec.suspend=false` to properly test the fix

Reply:
added